src/wrappers/express.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable prefer-rest-params */
+/**
+ * @fileoverview Handlers for Express instrumentation
+ */
+
+const shimmer = require('shimmer');
+const tryRequire = require('../try_require.js');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const traceContext = require('../trace_context.js');
+const expressRunner = require('../runners/express.js');
+
+const express = tryRequire('express');
+
+
+/**
+ * Express requests middleware that runs in context
+ * @param {Request} req The Express's request data
+ * @param {Response} res The Express's response data
+ * @param {Function} next express function
+ */
+function expressMiddlewareContext(req, res, next) {
+    tracer.restart();
+    let expressEvent;
+    const startTime = Date.now();
+    try {
+        expressEvent = expressRunner.createRunner(req, startTime);
+        tracer.addRunner(expressEvent);
+    } catch (err) {
+        utils.debugLog(err);
+    }
+
+    // Inject trace functions
+    const { label, setError } = tracer;
+    req.epsagon = {
+        label,
+        setError,
+    };
+
+    next();
+    if (!req.route) {
+        return;
+    }
+
+    // Handle response
+    res.on('finish', function handleResponse() {
+        try {
+            expressRunner.finishRunner(expressEvent, this, req, startTime);
+        } catch (err) {
+            tracer.addException(err);
+        }
+        tracer.sendTrace(() => {});
+    });
+}
+
+
+/**
+ * Express requests middleware
+ * @param {Request} req The Express's request data
+ * @param {Response} res The Express's response data
+ * @param {Function} next express function
+ */
+function expressMiddleware(req, res, next) {
+    // Initialize tracer
+    const tracerObj = tracer.createTracer();
+    tracer.getTrace = traceContext.get;
```

Comment:
You should set the get method in initialization, not execution

Reply:
Done